torch/testing/_internal/composite_compliance.py

```diff
@@ -187,11 +307,4 @@ def wrap(e):
         with enable_python_mode(CompositeCompliantTensor):
             op(*args, **kwargs)
     except RuntimeError as err:
-        raise RuntimeError("CompositeImplicitAutograd compilance check failed with "
-                           "the above error. If you are adding an OpInfo of an "
-                           "existing operator, please feel free to skip this test "
-                           "because the problem was pre-existing and file an issue. "
-                           "Otherwise, if you added a new operator, please read "
-                           "through the CompositeImplicitAutograd Compliance section in "
-                           "aten/src/ATen/native/README.md for how to resolve this. "
-                           ) from err
+        raise_composite_compliance_error(err)
```

Comment:
How long do the new tests take to run?

Reply:
Total runtime is 1:14 on both CPU and CUDA.
CPU-only is: 32s